### ISSUE TITLE
Unknown parameters : projectid during scale up VM by project admin/user in 4.17.1

### ISSUE THREAD

**Issue by levindecaro** [apache/cloudstack#6820]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
cloudstack-management
~~~

##### CLOUDSTACK VERSION

~~~
4.17.1
~~~

##### CONFIGURATION

kvm
SharedMountPoint (CephFS)

##### OS / ENVIRONMENT

CentOS Linux release 7.9.2009 (Core) 3.10.0-1160.45.1.el7.x86_64
libvirt-4.5.0-36.el7_9.5.x86_64
qemu-kvm-ev-2.12.0-44.1.el7_8.1.x86_64
qemu-img-ev-2.12.0-44.1.el7_8.1.x86_64
qemu-kvm-common-ev-2.12.0-44.1.el7_8.1.x86_64
libvirt-daemon-driver-qemu-4.5.0-36.el7_9.5.x86_64

##### SUMMARY

After upgrade from 4.16.1 to 4.17.1, project admin/user cannot scale up/down the VM instance reside in same project. However, use root admin could able to scale up and down.

##### STEPS TO REPRODUCE

~~~
1. Login as user of the project
2. Switch to the Project View
3. Shutdown the VM being scale up
4. Click the VM and click Scale up and select the target offering
5. Error Occur
6. Logout, and login with root admin
7. Switch to the project view
8. Click the VM and click Scale up and select the target offering
9. VM scaled successfully.

~~~

##### EXPECTED RESULTS

~~~
Project users able to scale up/down different offering in project scope.
~~~

##### ACTUAL RESULTS

Error throw out from UI `No permission to migrate volume, only root admin can migrate a volume`

<img width="439" alt="image" src="https://user-images.githubusercontent.com/36956864/195052647-afa1237c-3ab0-4746-8b91-e7bc0a77991d.png">

~~~

2022-10-11 17:16:51,651 DEBUG [c.c.a.ApiServlet] (qtp775386112-7759:ctx-94c69cf7) (logid:16180b81) ===START===  0:0:0:0:0:0:0:1 -- GET  id=c0bab1bb-0e83-4f82-a18c-9ababfc1276e&serviceofferingid=0161471b-4854-46cd-ace3-538ce398775e&automigrate=false&command=scaleVirtualMachine&response=json&projectid=342e988f-e70f-4133-ab13-2a37eabae492
2022-10-11 17:16:51,658 DEBUG [c.c.a.ApiServer] (qtp775386112-7759:ctx-94c69cf7 ctx-2948ad8f) (logid:16180b81) CIDRs from which account 'Acct[7ca3efa2-a7eb-4d6a-ad0f-f6a46de66b66-cloudstack.admin] -- Account {"id": 14, "name": "cloudstack.admin", "uuid": "7ca3efa2-a7eb-4d6a-ad0f-f6a46de66b66"}' is allowed to perform API calls: 0.0.0.0/0,::/0
2022-10-11 17:16:51,671 DEBUG [c.c.u.AccountManagerImpl] (qtp775386112-7759:ctx-94c69cf7 ctx-2948ad8f) (logid:16180b81) Access to Acct[78b5366c-69fa-4314-8579-19ced81d279c-PrjAcct-GPA - GPIS-2] -- Account {"id": 175, "name": "PrjAcct-GPA - GPIS-2", "uuid": "78b5366c-69fa-4314-8579-19ced81d279c"} granted to Acct[7ca3efa2-a7eb-4d6a-ad0f-f6a46de66b66-cloudstack.admin] -- Account {"id": 14, "name": "cloudstack.admin", "uuid": "7ca3efa2-a7eb-4d6a-ad0f-f6a46de66b66"} by DomainChecker
2022-10-11 17:16:51,673 DEBUG [c.c.u.AccountManagerImpl] (qtp775386112-7759:ctx-94c69cf7 ctx-2948ad8f) (logid:16180b81) Access to VM[User|gpis-application-server] granted to Acct[7ca3efa2-a7eb-4d6a-ad0f-f6a46de66b66-cloudstack.admin] -- Account {"id": 14, "name": "cloudstack.admin", "uuid": "7ca3efa2-a7eb-4d6a-ad0f-f6a46de66b66"} by DomainChecker
2022-10-11 17:16:51,674 WARN  [c.c.a.d.ParamGenericValidationWorker] (qtp775386112-7759:ctx-94c69cf7 ctx-2948ad8f) (logid:16180b81) Received unknown parameters for command scaleVirtualMachine. Unknown parameters : projectid
2022-10-11 17:16:51,694 DEBUG [o.a.c.f.j.i.AsyncJobManagerImpl] (qtp775386112-7759:ctx-94c69cf7 ctx-2948ad8f) (logid:16180b81) submit async job-68509, details: AsyncJobVO: {id:68509, userId: 12, accountId: 14, instanceType: VirtualMachine, instanceId: 3954, cmd: org.apache.cloudstack.api.command.user.vm.ScaleVMCmd, cmdInfo: {"serviceofferingid":"0161471b-4854-46cd-ace3-538ce398775e","response":"json","ctxUserId":"12","httpmethod":"GET","ctxStartEventId":"147487","id":"c0bab1bb-0e83-4f82-a18c-9ababfc1276e","ctxDetails":"{\"interface com.cloud.offering.ServiceOffering\":\"0161471b-4854-46cd-ace3-538ce398775e\",\"interface com.cloud.vm.VirtualMachine\":\"c0bab1bb-0e83-4f82-a18c-9ababfc1276e\"}","projectid":"342e988f-e70f-4133-ab13-2a37eabae492","ctxAccountId":"14","uuid":"c0bab1bb-0e83-4f82-a18c-9ababfc1276e","automigrate":"false","cmdEventType":"VM.UPGRADE"}, cmdVersion: 0, status: IN_PROGRESS, processStatus: 0, resultCode: 0, result: null, initMsid: 52235250599, completeMsid: null, lastUpdated: null, lastPolled: null, created: null, removed: null}
2022-10-11 17:16:51,695 DEBUG [c.c.a.ApiServlet] (qtp775386112-7759:ctx-94c69cf7 ctx-2948ad8f) (logid:16180b81) ===END===  0:0:0:0:0:0:0:1 -- GET  id=c0bab1bb-0e83-4f82-a18c-9ababfc1276e&serviceofferingid=0161471b-4854-46cd-ace3-538ce398775e&automigrate=false&command=scaleVirtualMachine&response=json&projectid=342e988f-e70f-4133-ab13-2a37eabae492

~~~

**Comment by weizhouapache** [comment]
this looks same as #6701

**Comment by rohityadavcloud** [comment]
cc @shwstppr @harikrishna-patnala

**Comment by weizhouapache** [comment]
closed via #6704
